examples/problems/co2ptflashproblem.hh

```diff
@@ -498,6 +498,24 @@ private:
             fs.setViscosity(FluidSystem::gasPhaseIdx, FluidSystem::viscosity(fs, paramCache, FluidSystem::gasPhaseIdx));
         }
 
+        // determine the component total fractions
+        // TODO: duplicated code here, while should be refactored out when we swithing
+        // to starting from total mole fractions
+        Dune::FieldVector<Scalar, numComponents> z(0.0);
+        Scalar sumMoles = 0.0;
+        for (unsigned phaseIdx = 0; phaseIdx < numPhases; ++phaseIdx) {
+            for (unsigned compIdx = 0; compIdx < numComponents; ++compIdx) {
+                Scalar tmp = Opm::getValue(fs.molarity(phaseIdx, compIdx) * fs.saturation(phaseIdx));
+                z[compIdx] += Opm::max(tmp, 1e-8);
```

Comment:
We're inside the `Opm` namespace, so you probably should not use the `Opm::` namespace prefix here.  Alternatively, you should write these as `::Opm::getValue(...)` and `... += ::Opm::max(...)` instead.

On a second note, `fs.saturation(phaseIdx)` is (hopefully) constant for each `compIdx`, so you could arguably write
```C++
for (phaseIdx) {
    const auto s = getValue(fs.saturation(phaseIdx))
    for (compidx) {
        const auto tmp = getValue(fs.molarity(phaseIdx, compIdx)) * s;
        ...
    }
}
```
instead.

Reply:
good catch! done fixing.